app/src/main/java/nl/tudelft/b_b_w/model/GetDatabaseHandler.java

```diff
@@ -77,7 +77,7 @@ public final String getContactName(String hash) {
         cursor.close();
 
 
-        return (block.getSequenceNumber()==1) ? block.getOwner() : block.getOwner()+"'s friend #" + (block.getSequenceNumber() - 1);
+        return (block.getSequenceNumber()==1) ? block.getOwner() : block.getOwner()+"'s friend #" + block.getSequenceNumber();
```

Comment:
This should be (block.getSequenceNumber() - 1) because the first block is always yourself and the next block (block with sequence number 2) is then your first friend. So actually it's the second block but its your first friend.